gazelle/manifest/defs.bzl

```diff
@@ -68,56 +77,72 @@ def gazelle_python_manifest(
         requirements = name + "_requirements_gen"
 
     update_args = [
-        "--manifest-generator-hash",
-        "$(rootpath {})".format(manifest_generator_hash),
-        "--requirements",
-        "$(rootpath {})".format(requirements),
-        "--pip-repository-name",
-        pip_repository_name,
-        "--modules-mapping",
-        "$(rootpath {})".format(modules_mapping),
-        "--output",
-        "$(rootpath {})".format(manifest),
-        "--update-target",
-        update_target_label,
+        "--manifest-generator-hash=$(location {})".format(manifest_generator_hash),
```

Comment:
Please use `rootpath` in favour of `location` here and elsewhere. See https://bazel.build/reference/be/make-variables for more context, but generally, don't use `location`.

Reply:
There were seven `location`s; I was able to make four into `rootpath` and three into `execpath`.

Reply:
Perfect!